plugins/modules/dcnm_vrf.py

```diff
@@ -1173,8 +1363,92 @@ def push_to_remote(self, is_rollback=False):
                         return
                     self.failure(resp)
 
-        method = 'POST'
         if self.diff_attach:
+            for d_a in self.diff_attach:
+                for v_a in d_a['lanAttachList']:
+                    if v_a.get('vrf_lite'):
+
+                        '''Before apply the vrf_lite config, need double check if the switch role is started wth Border'''
+
+                        if (self.role.lower() != 'border' and
```

Comment:
Same comment here about using a regular expression to check for role `border` in one line instead of lines 1373-1378

Reply:
done.